### PR TITLE
Feature/europe pmc updates

### PR DESCRIPTION
**Pull request by jweiser**
updates the description of the Reactome project and the URLs we are exporting to Europe PMC (content page instead of pathway browser)

### CODE REVIEW

File: data-exporter/pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	<groupId>org.reactome.release</groupId>
 	<artifactId>data-exporter</artifactId>
 	<packaging>jar</packaging>
-	<version>1.0-SNAPSHOT</version>
+	<version>1.1.0-SNAPSHOT</version>
```
Comment (cookersjs) [inline data-exporter/pom.xml]: We should probably start moving away from the string `SNAPSHOT` being in our versioning, all my new repos have it as well

File: data-exporter/src/main/java/org/reactome/release/dataexport/EuropePMC.java [hunk 2]
```diff
@@ -124,10 +124,8 @@ private String getEuropePMCProfileXML() {
 		nodeNameToValue.put("id", Integer.toString(reactomeProviderID));
 		nodeNameToValue.put("resourceName", "Reactome");
 		nodeNameToValue.put("description",
-			"Reactome is a free, open-source, curated and peer-reviewed pathway database. " +
-			"Our goal is to provide intuitive bioinformatics tools for the visualization, " +
-			"interpretation and analysis of pathway knowledge to support basic research, genome analysis, " +
-			"modeling, systems biology and education.");
+			"Reactome is a free, open-source, curated and peer-reviewed pathway database."
```
Comment (cookersjs) [inline data-exporter/src/main/java/org/reactome/release/dataexport/EuropePMC.java]: Did we lose this the mission statement?

File: data-exporter/src/test/java/org/reactome/release/dataexport/EuropePMCTest.java [hunk 3]
```diff
@@ -47,7 +47,7 @@ public void singleEuropePMCLinkFromMockGraphDB() {
 	public void correctEuropePMCLinkXMLElement() throws ParserConfigurationException {
 		final String PATHWAY_DISPLAY_NAME = "p53-Dependent G1 DNA Damage Response";
 		final String PATHWAY_STABLE_ID = "R-HSA-69563";
-		final String PATHWAY_URL = "https://www.reactome.org/PathwayBrowser/#/" + PATHWAY_STABLE_ID;
+		final String PATHWAY_URL = "https://reactome.org/content/detail/" + PATHWAY_STABLE_ID;
```
Comment (cookersjs) [inline data-exporter/src/test/java/org/reactome/release/dataexport/EuropePMCTest.java]: Could this become `ReactomeURLConstants.CONTENT_DETAIL_URL + PATHWAY_STABLE_ID`?

File: data-exporter/pom.xml (hunk 1 above)
Comment (jweiser) [reply data-exporter/pom.xml]: Yeah, I'll remove it seeing is this is a finished minor update.  I had to refresh my memory on when to use SNAPSHOT and when not to use it.

https://stackoverflow.com/questions/5901378/what-exactly-is-a-maven-snapshot-and-why-do-we-need-it

File: data-exporter/src/main/java/org/reactome/release/dataexport/EuropePMC.java (hunk 2 above)
Comment (jweiser) [reply data-exporter/src/main/java/org/reactome/release/dataexport/EuropePMC.java]: No, we still have this full mission statement but we decided to go with a shorter version for use in Europe PMC.

File: data-exporter/src/test/java/org/reactome/release/dataexport/EuropePMCTest.java (hunk 3 above)
Comment (jweiser) [reply data-exporter/src/test/java/org/reactome/release/dataexport/EuropePMCTest.java]: Yeah, I think it can.  I guess I'm just used to literals in tests!

**Pull request merged**